forge-game/src/main/java/forge/game/ability/AbilityUtils.java

```diff
@@ -3200,6 +3206,11 @@ public static int doXMath(final int num, final String operators, final Card c, C
             return num * -1;
         } else if (s[0].contains("Times")) {
             return num * secondaryNum;
+        } else if (s[0].contains("DivideEvenlyUp")) {
+            if (secondaryNum == 0) {
+                return 0;
+            }
+            return num / secondaryNum + (num % secondaryNum == 0 ? 0 : 1);
```

Comment:
use Math.ceil?

`(int)Math.ceil((double)num / secondaryNum);`

Reply:
There are claims that it can lead to bad results:
https://stackoverflow.com/questions/7139382/java-rounding-up-to-an-int-using-math-ceil